themes/odin-theme/static/css/style.css

```diff
@@ -1,18 +1,18 @@
 ::-webkit-selection {
-  background-color: #37E;
-  color: #333;
+  background-color: #D5824F;
```

Comment:
I do not like this this orange highlight. Please revert to the original colours.